sql/parser/eval.go

```diff
@@ -1246,8 +1257,7 @@ func ParseDate(s DString) (DDate, error) {
 	if err != nil {
 		return DummyDate, err
 	}
-
-	return MakeDDate(t), nil
+	return DDate{Time: t}, nil
```

Comment:
The time.Parse() documentation says, "In the absence of a time zone indicator, Parse returns a time in UTC."